### PR TITLE
Fb ignore missing objects

### PR DESCRIPTION
**Pull request by alonbrody**
When an error code 100 arrives it means that a specific object is missing. We can skip it in case that the user wanted to skip the missing/deleted objects.

@alonweissfeld @liorrozen

### CODE REVIEW

**Comment by alonbrody** [comment]
@alonweissfeld @liorrozen Same PR but without the changes of the previous [PR](https://github.com/panoplyio/facebook-insight-stream/pull/7) that was merged

File: index.js [hunk 1]
```diff
@@ -314,12 +316,15 @@ FacebookInsightStream.prototype.handleError = function ( error, retry ) {
...
-function SkipedError ( error ) {
+function SkippedError ( error ) {
     return error.skip === true;
 }
 
-function errorHandler ( body )  {
+function errorHandler ( options, body )  {
     if ( body.error ) {
+        // Missing data is flagged by the error code 100
+        body.error.skip = options.ignoreMissing && body.error.code === 100
```
Comment (liorrozen) [inline index.js]: Please convert this error code to a constant and add a link to the relevant documentation page referencing the error - https://developers.facebook.com/docs/marketing-api/error-reference

Also I think that the comment should use the same phrasing as the documentation. Error 100 means "missing parameters". If i understand it correctly, this doesn't mean that there is an object missing - rather that the request is missing some parameters.

**Comment by liorrozen** [comment]
@alonbrody - no further comments.
@alonweissfeld - you're up!

**Comment by alonweissfeld** [comment]
@alonbrody I agree with @liorrozen on the constant and documentation. Other than that - looks good.

**Comment by alonbrody** [comment]
@alonweissfeld @liorrozen changed. Note that I couldn't find the documentation about this error in FB docs so I've placed the actual error I received there.

**Comment by alonweissfeld** [comment]
@alonbrody Looks good - @liorrozen If you approve you can merge.

**Comment by liorrozen** [comment]
@alonbrody @alonweissfeld - done and done. Merging.

**Pull request merged**